packages/api/cms-api/src/dam/files/files.controller.ts

```diff
@@ -90,6 +94,11 @@ export function createFilesController({ Scope: PassedScope }: { Scope?: Type<Dam
             @GetCurrentUser() user: CurrentUser,
             @Headers("range") range?: string,
         ): Promise<void> {
+            if (process.env.NODE_ENV) {
+                // This workaround is only necessary in dev mode. In deployed envs, the preview URLs are delivered through the same URL as the admin app (because of the auth proxy).
```

Comment:
This works in our deployment setup, but doesn't work in other cases. Can't we solve that differently? I'm not really fond of this if.

@max-debug022 do you have any ideas?

Reply:
As discussed internally, the preferable solution is setting the correct CSP-Config when initializing helmet in main.ts